app/subscriber/src/features/utils/navigateAndScroll.ts

```diff
@@ -0,0 +1,7 @@
+export const navigateAndScroll = (navigate: any, path: string) => {
+  navigate(path);
+  const cont = document.getElementsByClassName('contents-container')[0];
```

Comment:
Should we use a `ref` instead of DOM search?

Reply:
ref may improve maintainability so we don't rely on `className` - not a big deal though

Reply:
The frame that need to be scrolled is into the Landing file, for that the hierarchy needs to be built up.
Since it's just a scroll into the DOM in two specific situations, I was talking to @jdtoombs about it, and we are ok, since it would involver bigger changes.

Reply:
Ah so it's way too far up to get a good reference on it.  One of the headaches with React.

Reply:
Unfortunately ... yes ... :(